packages/mongo/collection.js

```diff
@@ -92,7 +96,15 @@ Mongo.Collection = function Collection(name, options) {
   else if (Meteor.isClient) this._connection = Meteor.connection;
   else this._connection = Meteor.server;
 
-  if (!options._driver) {
+  if (options._driver) {
+    if (typeof options._driver.open !== 'function') {
+      throw new Error('If you are creating the driver manually using new ' +
+        'MongoInternals.RemoteCollectionDriver then you need to use ' +
+        'Promise.await() or await on it. Since it is async in recent ' +
+        'versions of Meteor. ' +
+        'Read more https://docs.meteor.com/changelog.html.');
```

Comment:
Sorry for a trivial grammar nitpick!
```suggestion
      throw new Error('If you are creating the driver manually using new ' +
        'MongoInternals.RemoteCollectionDriver then you need to use ' +
        'Promise.await() or await on it since it is async in recent ' +
        'versions of Meteor. ' +
        'Read more: https://docs.meteor.com/changelog.html.');
```